client/src/Admin/SitesForm.js

```diff
@@ -10,14 +10,14 @@ function SitesForm() {
   const navigate = useNavigate();
   const user = useAuthContext();
   const { id } = useParams();
-  const [partners, setPartner] = useState(null);
-  const [selectedPartner, setSelectedPartner] = useState(null);
+  const [partners, setPartners] = useState(null);
   const [data, setData] = useState({
     name: '',
     address: '',
-    phonenumber: '',
+    phoneNumber: '',
     email: '',
     website: '',
+    NutritionPartnerIds: [],
```

Comment:
I've added an array property to store the ids of associated NutritionPartner records